app/templates/components/workflow-files.hbs

```diff
@@ -1,62 +1,62 @@
 {{yield}}
 <style>
-.table-text {
-  display: inline-block;
-  overflow: hidden;
-  white-space: pre-wrap;
-  width:200px;
-}
+  .table-text {
+    display: inline-block;
+    overflow: hidden;
+    white-space: pre-wrap;
+    width: 200px;
+  }
 </style>
 <p>
-   Attaching manuscript/article files to this submission.
+  Attach manuscript/article files to this submission.
 </p>
-    <div class="form-group row">
-      <div class="col-md-12">
-          <input type="file" id="file-multiple-input" multiple size="50" onchange={{action "getFiles"}}  class="text-gray-300" style="border-width:2px; cursor:pointer; border-style: dashed; padding:30px; width:100%" >
-      </div>
-    </div>
-    <div class="form-group row">
-      <div class="col-md-12">
-        {{!-- <textarea id="textarea-input" name="textarea-input" rows="3" class="form-control" placeholder="Description"></textarea> --}}
-      </div>
-    </div>
+<div class="form-group row">
+  <div class="col-md-12">
+    <input type="file" id="file-multiple-input" multiple size="50" onchange= {{action "getFiles"}} class="text-gray-300" style="border-width:2px; cursor:pointer; border-style: dashed; padding:30px; width:100%">
+  </div>
+</div>
+<div class="row">
 
+  <div class="col-lg-12">
+    <table class="table table-responsive-sm table-sm">
+      <thead>
+        <tr class="text-center">
+          <th>File Name</th>
+          <th>Manuscript</th>
+          <th>Description</th>
+          <th>Action</th>
+        </tr>
+      </thead>
+      <tbody>
+        {{#each files as |file|}}
+          <tr>
+            <td>
+              <p class="table-text">{{file.name}}</p>
+            </td>
+            <td class="text-center">
+              <select onchange= {{action (mut file.fileRole) value="target.value"}}>
+                <option value="supplement" selected={{eq file.fileRole "supplement"}}>Supplement</option>
+                <option value="manuscript" selected={{eq file.fileRole "manuscript"}}>Manuscript</option>
+                <option value="table" selected={{eq file.fileRole "table"}}>Table</option>
+                <option value="figure" selected={{eq file.fileRole "figure"}}>Figure</option>
```

Comment:
Maybe we should add an other option here as well so we cover everything?

Reply:
@htpvu wanted it to be ENUM-style to purposefully limit the options, I think. Plus, supplement is basically "other".